introduction.md

```diff
@@ -2,4 +2,10 @@ This is an introduction to my course "Mastering GitGub"
 
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?